src/App.scss

```diff
@@ -1 +1,41 @@
-// styles go here
+.post {
```

Comment:
these styles shouldn't be on App.scss
(should be different blocks for post, comments, user etc - styles to every component should be on a separate file)